Assets/Scripts/Scenes/InGame/Stage/StageTileTest.cs

```diff
@@ -6,9 +6,10 @@ namespace Scenes.Ingame.Stage
 {
     public class StageTileTest : MonoBehaviour
     {
-        private float _Test = 30;
+        private float _Test;
```

Comment:
```suggestion
        private float _test;
```
細かいですが_の次は小文字です。また機能が完成しているなら、変数名をtestから適切なものに変えてほしいです

Reply:
動作確認するためだけのスクリプトだったので削除しておきます